### DISCUSSION TITLE
【Check Gradients in MLP Projector】

### DISCUSSION THREAD

**Discussion by Luo-Z13** [haotian-liu/LLaVA#1714] [Q&A]
Does anyone know how to check the gradients on the MLP projector during the pre-training phase of llava, or how to view the process of gradient backpropagation? I tried adding a projector, but it seems that the loss is not decreasing during training.
